modules/gdscript/language_server/gdscript_extend_parser.cpp

```diff
@@ -687,6 +687,11 @@ String ExtendGDScriptParser::get_text_for_lookup_symbol(const LSP::Position &p_c
 	return longthing;
 }
 
+_FORCE_INLINE_ bool is_identifier_char(char32_t p_char) {
+	// Unicode ID_CONTINUE is always a superset of ID_START as per https://www.unicode.org/reports/tr31/
+	return is_ascii_identifier_char(p_char) || is_unicode_identifier_continue(p_char);
```

Comment:
`is_ascii_identifier_char` is redundant, it is subset of `is_unicode_identifier_continue`.